numpy/linalg/linalg.py

```diff
@@ -1382,7 +1382,7 @@ def cond(x, p=None):
 
     Parameters
     ----------
-    x : (M, N) array_like
+    x : (..., M, N) array_like
```

Comment:
> Google it ;)

I'm not sure what you're suggesting to google...

The numpy.linalg.cond function is documented as "The condition number of x is defined as the norm of x times the norm of the inverse of x." so should this be updated if `cond` does something else when the matrix is not square?